### ISSUE TITLE
Appimage version is not responding (pop os and intel UHD Graphic)

### ISSUE THREAD

**Issue by dheerapat** [bjornbytes/lovr#639]
I allowed permission on the app and I try to run by clicking it first but it not response so I use terminal to run it and encounter this error:
```
Error [GENERAL |  | OpenXR-Loader] : RuntimeManifestFile::FindManifestFiles - failed to determine active runtime file path for this environment
Error [GENERAL | xrEnumerateInstanceExtensionProperties | OpenXR-Loader] : RuntimeInterface::LoadRuntimes - unknown error
Error [GENERAL | xrEnumerateInstanceExtensionProperties | OpenXR-Loader] : RuntimeInterface::LoadRuntimes - failed to load a runtime
Error [GENERAL | xrEnumerateInstanceExtensionProperties | OpenXR-Loader] : Failed to find default runtime with RuntimeInterface::LoadRuntime()
Error [GENERAL | xrEnumerateInstanceExtensionProperties | OpenXR-Loader] : Failed querying extension properties
Error:
GPU error: Failed to load vulkan library

Stack:
boot.lua:90: in function <boot.lua:4>
[C]: in function 'xpcall'
Segmentation fault (core dumped)
```
My system is intel i3 gen 10 with UHD graphic no dGPU and running pop os

**Comment by bjornbytes** [comment]
Thanks for the issue.  I believe this was already fixed in 64591c02ed7ccba96676c846c7378d0deb8cdf01 which will be included in the next release.  You can try using a build from GitHub actions which includes the fix, or create a symlink for `libvulkan.so`.  Let me know if you're still having issues.

**Comment by dheerapat** [comment]
Update: I install Vulkan SDK for my OS and your program works like a charm. All resource is [here](https://vulkan.lunarg.com/sdk/home)
